lib/modules/commentNavigator.js

```diff
@@ -214,7 +222,11 @@ module.go = () => {
 	});
 
 	if (module.options.showByDefault.value) {
-		commentNav();
+		if (module.options.autoNavigateByIama.value && isCurrentSubreddit('IAmA')) {
+			showNavigator('IAmA');
```

Comment:
In order for `IAmA` to be set as initial also when  `showByDefault` enabled, the initial value of `currentCategory` can be set to `isCurrentSubreddit('IAmA') ? 'IAmA' : '';`.

Reply:
Will do, but does that matter? Doesn't the user need to click on one of the "navigate by" buttons to show the navigator if it isn't shown by default? I don't use the feature much, so I might be missing something.

Reply:
It can also be opened by keyboardNav (shortcut `n`).